Article/Article.js

```diff
@@ -1,29 +1,42 @@
-// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoisted you will need to start your 
+//code at the bottom of the page.  Look for the comment "START 
+//HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton = this.domElement.querySelector('.expandButton');
+    // Using your expandButton reference, update the text on
+    // your expandButton to say "expand"
+    this.expandButton.textContent = 'expand';
+    // Set a click handler on the expandButton reference, 
+    //calling the expandArticle method.
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
This looks great. I would suggest using the es6 syntax instead of the this keyword. Great job though.